### ISSUE TITLE
[Website] Blog-Community Call- Youtube Video: <iframe> and <frame> elements have not an accessible name.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6437]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/blog/2019/Community-Call-December/ 9/14/2021

### Application Name

Website

### Problem Description

[ADO 36108480](https://microsoft.visualstudio.com/OS/_workitems/edit/36108480)

Repro Steps:
1. Launch the above URL.
2. Run Accessibility Insight Tool of Web.
3. Observe the issue.

Actual Result:
`<iframe>` and `<frame>` elements have not an accessible name.

Expected Result:
Ensure `<iframe>` and `<frame>` elements have an accessible name.

### Card JSON

```JSON
No card
```

**Comment by paulcam206** [comment]
`<iframe>`s in our blog posts should be specifying the `title` attribute (see [here](https://accessibilityinsights.io/info-examples/web/frame-title/))

**Comment by paulcam206** [comment]
@JeanRoca @kat-y 👀

**Comment by PankajBhojwani** [comment]
Turns out there were more blog posts that have this issue than just the 1 fixed in the PR, reopening this
